staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -207,7 +207,7 @@ func newAdmissionMetrics() *AdmissionMetrics {
 			Namespace:      namespace,
 			Subsystem:      subsystem,
 			Name:           "webhook_fail_open_count",
-			Help:           "Admission webhook fail open count, identified by name and broken out for each admission type (validating or mutating).",
+			Help:           "Admission webhook fail open count, identified by name and broken out for each admission type (validating or admit).",
```

Comment:
We can see that the `type` label is "admit" here: https://github.com/modulitos/kubernetes/blob/80552926be0dd67ff2607ccce095a1d64b8775c7/staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go#L211